.gitignore

```diff
@@ -53,3 +53,6 @@ docs/_build/
 # node.js dependencies 
 node_modules/
 
+# Libraries directory
+lib/
```

Comment:
Why add to the gitignore? We have deleted it, right?

Reply:
Just in case you accidentally commit them.